app/Filament/Pages/Installer/PanelInstaller.php

```diff
@@ -32,54 +34,43 @@ class PanelInstaller extends SimplePage implements HasForms
 {
     use CheckMigrationsTrait;
     use EnvironmentWriterTrait;
-    use HasUnsavedDataChangesAlert;
     use InteractsWithForms;
 
     public $data = [];
 
     protected static string $view = 'filament.pages.installer';
 
+    private User $user;
+
     public function getMaxWidth(): MaxWidth|string
     {
         return MaxWidth::SevenExtraLarge;
     }
 
-    public static function show(): bool
+    public static function isInstalled(): bool
     {
-        if (User::count() <= 0) {
-            return true;
-        }
-
-        if (config('panel.client_features.installer.enabled')) {
-            return true;
-        }
-
-        return false;
+        // This defaults to true so existing panels count as "installed"
+        return env('APP_INSTALLED', true);
```

Comment:
Should be using config instead of env.

Reply:
We can revisit this. For now i'm going to merge it so the installer isn't broken for every new install.